safe/gui/widgets/test/test_field_mapping_tab.py

```diff
@@ -0,0 +1,105 @@
+# coding=utf-8
+"""Test Field Mapping Tab."""
+
+import unittest
+
+from safe.test.utilities import get_qgis_app, load_test_vector_layer
+
+QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
+
+from safe.gui.widgets.field_mapping_tab import FieldMappingTab
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->